hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -6,7 +6,7 @@ parameters:
 - name: IMAGE_TAG
   required: true
 - name: NAMESPACE
-  value: openshift-monitoring
+  value: openshift-route-monitor-operator
```

Comment:
@NautiluX this is where we configure the namespace, it's just easier to manage and we can move it then later using the pipelines